README.md

```diff
@@ -141,12 +141,37 @@ build_windows_cuda.bat
 Build can take ~2 hours depending on the machine configuration.
 
 ### Build and test using docker
+
+### Prerequisites
+* Ensure virtualization is enabled in both the BIOS settings of your computer and the Windows virtualization feature
+* Install WSL 2 on your system
+* Use the command line to set WSL 2 as the default version by running the command: wsl --set-version 2
+* Install Ubuntu-18.04 from the Microsoft store
+* Enable Docker integration with WSL 2 (in Docker Desktop settings -> Resources -> WSL integration -> Enable Ubuntu-18.04 -> Apply&restart)
```

Comment:
we never instructed to install docker ?

Reply:
i will add it

Reply:
added the instruction and linked related document